### PR TITLE
Amend LICENSE

### PR DESCRIPTION
**Pull request by mattrobenolt**
/cc @dcramer 

Refs: #19

### CODE REVIEW

**Comment by dcramer** [comment]
:+1:

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Copyright (c) 2015 Functional Software, Inc and individual contributors.
```
Comment (tianon) [inline LICENSE]: Probably want to adjust `Docker, Inc.` down below in clause 3 as well, right?

**Comment by dcramer** [comment]
Yeah IDK whats most correct, but you can just use Sentry instead of Functional Software Inc, and replace Docker in that clause. Just maintain their copyright clause and it should be ok.

**Comment by mattrobenolt** [comment]
@dcramer Good now?

**Comment by dcramer** [comment]
yep

**Pull request merged**